resultsdbupdater/utils.py

```diff
@@ -184,64 +184,45 @@ def cips_post_to_resultsdb(msg):
     session = retry_session()
     # define variables from cips.json
     msg_id = msg['headers']['message-id']
-    ci_type = msg['headers']['ci_type']
     component = msg['headers']['component']
-    brew_task_id = msg['headers']['brew_task_id']
     msg_body = msg['body']['msg']
-    tests = msg_body['tests']
-
-    arch = msg_body['environment']['arch']
-    brew_tag = msg_body['environment']['brew_tag']
-    build_type = msg_body['environment']['build_type']
-
-    jenkins_job_url = msg_body['infrastructure']['jenkins_job_url']
-    jenkins_build_url = msg_body['infrastructure']['jenkins_build_url']
-
-    cips_report = msg_body['results']['cips_report']
-    cips_status = msg_body['results']['cips_status']
-
-    testcase_url = jenkins_job_url
+    msg_body_ci = msg['body']['msg']['ci']
+    msg_body_artifact = msg['body']['msg']['artifact']
+    url = msg_body['run']['url'].rsplit('/', 3)[0]
+    outcome = msg_body['status']
 
     # variables to be passed to create_result
     groups = [{
         'uuid': str(uuid.uuid4()),
-        'ref_url': jenkins_build_url
+        'url': url
```

Comment:
This should be URL the Jenkins run, so it should be the value of `msg_body['run']['url']`